### ISSUE TITLE
Would it be good to have `setOperationFlag('normalize', false)`

### ISSUE THREAD

**Issue by zhujinxuan** [ianstormtaylor/slate#1544]
#### Do you want to request a *feature* or report a *bug*?

features;

#### What's the current behavior?

In transforming node from one type to another type, or in customizing methods of delete/paste fragment at range, `{normalize: false}` is often applied until the final stage.  Because in the middle of the operations(for example, set type in a nested chain of blocks), normalization is often bad. 

However, writing `{normalize: false}` everywhere is a bit wordy. Would you think whether it is good to have a PR about `setOperationFlag('normalize', false)`?

**Comment by CameronAckermanSEL** [comment]
I've been working on this and hope to have something to look at today or by the end of the week
